relationships/synthesis/APM-APPLICATION-to-INFRA-ELASTICACHEREDIS.yml

```diff
@@ -0,0 +1,44 @@
+relationships:
```

Comment:
Following the `DATABASE` example, this file should be named `APM-APPLICATION-to-REDIS.yml`.